### DISCUSSION TITLE
https://thedatasavvycorner.com/blogs/17-quarto-comments

### DISCUSSION THREAD

**Discussion by giscus** [r-leyshon/blogging#70] [Announcements]
# https://thedatasavvycorner.com/blogs/17-quarto-comments

Comparing Hypothesis, Utterances & Giscus.

https://thedatasavvycorner.com/blogs/17-quarto-comments

**Comment by marianklose** [comment]
Thank you for this helpful blogpost :)

**Comment by zeigerpuppy** [comment]
Thanks for the summary, very useful.
Hypothes.is is a great this tool, but they really are not friendly to self-hosting.
See https://github.com/hypothesis/h/issues/4899 for discussion.

An intersting alternative is https://github.com/DanilaFe/matrix-highlight

**Reply by zeigerpuppy** [reply]
Oh, for simple commenting (no highlighting), Isso is really easy to self host and integrate.

https://isso-comments.de/

**Reply by r-leyshon** [reply]
Oh nice, thanks I will check these out. Simple commenting was what I needed for this blog, so I went for giscus but isso looks neat too.
